ytypes/node.go

```diff
@@ -195,18 +194,41 @@ func retrieveNodeContainer(schema *yang.Entry, root interface{}, path *gpb.Path,
 			return retrieveNode(cschema, fv.Interface(), util.TrimGNMIPathPrefix(path, p[0:to]), np, args)
 		}
 
-		for _, p := range schPaths {
-			if !util.PathMatchesPrefix(path, p) {
-				continue
+		args := args
+		switch args.reverseShadowPath {
+		case true:
+			schPaths := util.ShadowSchemaPaths(ft)
+			for _, p := range schPaths {
+				if !util.PathMatchesPrefix(path, p) {
```

Comment:
invert the polarity:
  if util.PathMatchesPrefix(path, p) {
    return checkPath(p, args)
  }

Reply:
Done for the 3 instances.